app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingPageUiState.kt

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.onboarding.view
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+
+/**
+ * Model containing data for the [OnboardingPage].
```

Comment:
Let's add the @param description.

Reply:
Think the params are quite self descriptive :) Similar to the comment here --> https://github.com/mozilla-mobile/fenix/pull/28376#discussion_r1061006455

Reply:
I ended up adding the param docs as it might be used by other callers 👍